Project.toml

```diff
@@ -4,13 +4,15 @@ version = "0.1.1"
 
 [deps]
 DynamicPolynomials = "7c1d4256-1411-5781-91ec-d7bc3513ac07"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
```

Comment:
Hm, should we make this an optional package? It is only used for one algorithm.

Reply:
I was also wondering that. On the other hand, ForwardDiff is pretty lightweight I think? If that's the case we could keep it as normal dependency (and maybe even have B&B as default one).

The derivative is also needed in the mean form enclosure (which can be used alone or together with B&B or together with MooreSkelboe)

Reply:
Since we'll compose BnB with the rest of the evaluation methods, I think it's ok to keep this as a dependency.